lib/todo_lint/cli.rb

```diff
@@ -6,9 +8,10 @@ class Cli
     # @example
     #   Cli.new([".rb", ".js"])
```

Comment:
the example should be updated to reflect the new interface

(this is the downsize of comments-as-documentation, it's very easy for them to go stale) (I might remove them) (but for now it should be updated)